package.json

```diff
@@ -59,11 +60,12 @@
     "passport-google-oauth": "^0.2.0",
     "passport-local": "^1.0.0",
     "react": "^0.13.3",
+    "react-router" : "^1.0.0-beta3",
     "react-helmet": "^1.0.1",
-    "react-router": "1.0.0-beta3",
```

Comment:
Just updated the onEnter transition. No thanks needed, your kit has made my life and I'm sure the lives of others much easier.